task/sast-coverity-check/0.3/migrations/0.3.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Created for task: sast-coverity-check@0.3
+# Creation time: $(date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Please replace this with real date value

Reply:
fixed,